src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/ProjectSetUnevaluatedUIPropertyValueAction.cs

```diff
@@ -25,9 +25,9 @@ public ProjectSetUnevaluatedUIPropertyValueAction(SetUnevaluatedUIPropertyValue
 
         protected override Task SetValueAsync(IProperty property)
         {
-            if (property is IEvaluatedProperty evaluatedProperty)
+            if (_parameter.Value == null)
             {
-                return evaluatedProperty.SetUnevaluatedValueAsync(_parameter.Value ?? string.Empty);
+                return property.DeleteAsync();
```

Comment:
Why don't we need to call `SetUnevaluatedValueAsync` here anymore?

Reply:
Ah, I assume this is https://github.com/dotnet/project-system/issues/6920#issuecomment-1101569461